stable/redmine/values.yaml

```diff
@@ -38,6 +38,12 @@ redmineLanguage: en
 # smtpPassword:
 # smtpTls:
 
+## Environment variables, to pass to the entry point
+##
+redmineEnvironment: []
```

Comment:
Because you wrap this in `if` in the template, you don't need to default to `[]`. See note above about 3850b04 as an example.